person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'Unkown', parent_permission: true)
+    @id = id
```

Comment:
I suggest you give the `id` a unique value. 👍 

For example
`@id = Random.rand(1..1000)`